### PR TITLE
T188435 Bring consistency to visual design of Interaction Timeline

### PR DESCRIPTION
**Pull request by prtksxna**
Per @nirzar, these proposed changes are only meant to change the styling and not the interactions or feature.

### CODE REVIEW

**Comment by prtksxna** [comment]
I am not very familiar with SCSS so please tell me what I'm doing wrong 😄 

There are some minimal design changes here, but only to accommodate the new colors. Some elements that were making the background 5% lighter on hover are now losing a hover state which is bad. I'd love suggestions on how we could avoid that, or if we could temporarily let it be as we move towards the [other updates](https://wikimedia.invisionapp.com/share/VRG4FI7NDJY#/screens/282441033).

File: client/src/components/timeline/date-list.js [hunk 1]
```diff
@@ -13,10 +13,10 @@ const DateList = ( { revisions } ) => {
 		prev = list;
 		return (
 			<div className="row day border pl-2 pr-2 pt-2 pb-2 mb-3" key={date.format( 'YYYY-MM-DD' )}>
-				<div className="col-xl-1 col-sm-2 col-12">
-					<Date icon="today" date={date.format( 'YYYY-MM-DD' )} />
+				<div className="col-xl-10 offset-xl-1 col-sm-8 col-12">
+					<Date icon="today" date={date.format( 'D MMMM Y' )} />
```
Comment (MusikAnimal) [inline client/src/components/timeline/date-list.js:17]: Will this format make sense in all regions?

Comment (Nikerabbit) [reply client/src/components/timeline/date-list.js:17]: Nope. Assuming this is using momentjs, then I recommend `LL` that picks an appropriate format for the locale that displays day, month and year.

Comment (davidbarratt) [reply client/src/components/timeline/date-list.js:17]: We originally had it that way, but it was changed in https://phabricator.wikimedia.org/T180725

Comment (prtksxna) [reply client/src/components/timeline/date-list.js:17]: Given that now the date has as much space as it needs would it make sense to switch back to `LL`?
screen shot 2018-05-14 at 11 01 46 am

Comment (mooeypoo) [reply client/src/components/timeline/date-list.js:17]: Do we have a good way for moment to be indicated the user's time prefs? I think moment does that on its own, but we do some magic in MediaWiki itself to push the chosen interface language.

In any case, I'm all for making the date more suitable for the user, so if LL works in moment, I'd say go with that.

Comment (davidbarratt) [reply client/src/components/timeline/date-list.js:17]: I think localizing the date format is the _proper_ thing to do, but again, this was a product decision, so a new ticket/discussion would have to happen (and ping Trevor). (or repoen https://phabricator.wikimedia.org/T180725)

**Comment by mooeypoo** [comment]
This requires rebasing; I tried locally and it works for me, but I had to resolve a couple of the conflicts blindly since they're changes to the html structure. Might be good if you can do that, @prtksxna ? it's only two conflicts, I just wasn't 100% sure my resolution between the two html structures is good.

**Comment by prtksxna** [comment]
I've made a huge mistake 😬Not sure how I managed to do this, I thought I was doing a simple rebase. Ignore this PR for now, I'll try to resolve this soon.

**Comment by prtksxna** [comment]
Going to split this into smaller changes.

**Pull request closed without merging**